flow/designs/asap7/mock-array-big/Element/io.tcl

```diff
@@ -41,8 +41,7 @@ proc zip {list1 list2} {
 
 foreach {direction direction2 names} $assignments {
     set mirrored [zip {*}$names]
-    set_io_pin_constraint -region $direction:* -pin_names [lindex $names 0]
-    set_io_pin_constraint -region $direction2:* -pin_names [lindex $names 1]
+    set_io_pin_constraint -region $direction:* -pin_names [lindex $names 0] -group -order
```

Comment:
You should use the `-group` and `-order` independently of the `-region`. I will update ppl with a warning message when they're used together.

Something like this should work as you expect:
`set_io_pin_constraint -region $direction:* -pin_names [lindex $names 0]`
`set_io_pin_constraint -pin_names [lindex $names 0] -group -order`

Reply:
I see. Though if this is a syntax error an error should be reported.